### DISCUSSION TITLE
Job requirements

### DISCUSSION THREAD

**Discussion by MurtadhaAlobaidi** [Hawkrin/KTH-IV1201-Recruitment-Application#74] [Ideas]
###  What qualifications and skills are necessary for a particular job? 🤔

 For example, a **software engineering** position might require a `degree in computer science`, proficiency in a `particular programming language`, and experience with `Agile development methodologies`.🤝 🧑‍🎓 💡
 
@Do you have any other ideas about what other skills are necessary for software engineering?

**Comment by MurtadhaAlobaidi** [comment]
test1

**Reply by Hawkrin** [reply]
We need to do it

**Comment by MurtadhaAlobaidi** [comment]
Testing 2

**Reply by Hawkrin** [reply]
Yes sir.

**Comment by MurtadhaAlobaidi** [comment]
testing 3

**Comment by Hawkrin** [comment]
TEST
